### PR TITLE
Added

### PR DESCRIPTION
**Pull request by mohak852**
---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by theswerd** [comment]
Please explain the point of this PR and link to the issue assigned to you

**Comment by mohak852** [comment]
@theswerd  Actually I have fixed #1350 this issue

**Comment by mohak852** [comment]
and I have fixed that solution but this problem is occurring only in this GitHub not on my local machine

**Comment by mohak852** [comment]
> This is unnecessary, you can just set the textAlign to center. Please revert changes
As asked by you I have centered the text you can check it now

**Comment by mohak852** [comment]
@theswerd

**Comment by theswerd** [comment]
Sorry for the confusion, the way to do this properly is set the textAlign of the ThemedText to TextAlign.center as seen in my comment

File: client/flutter/lib/pages/onboarding/legal_landing_page.dart [hunk 1]
```diff
@@ -36,12 +36,14 @@ class LegalLandingPage extends StatelessWidget {
...
-                    child: ThemedText(
-                      S.of(context).legalLandingPageTitle,
-                      variant: TypographyVariant.body,
-                      style: TextStyle(
-                        color: Constants.primaryColor,
-                        fontSize: 18,
+                    child: Center(
+                      child: ThemedText(
+                        S.of(context).legalLandingPageTitle,
+                        variant: TypographyVariant.body,
```
Comment (theswerd) [inline client/flutter/lib/pages/onboarding/legal_landing_page.dart:42]:
```suggestion
                       textAlign: TextAlign.center,
                        variant: TypographyVariant.body,
```

**Comment by mohak852** [comment]
@theswerd I have made changes mentioned by you and if this is correct merge it or if there any changes, please tell me to do so 


Thanks You

File: client/flutter/lib/pages/onboarding/legal_landing_page.dart (hunk 1 above)
Comment (mohak852) [reply client/flutter/lib/pages/onboarding/legal_landing_page.dart:43]: @theswerd I have made changes kindly go through that once.

**Pull request closed without merging**